ceph-releases/ALL/ubi9-ibm/__DOCKERFILE_BRANDING__

```diff
@@ -0,0 +1,12 @@
+ln -s /usr/share/ceph/mgr/dashboard/frontend/dist-ibm /usr/share/ceph/mgr/dashboard/frontend/dist && \
+sed -i \
+  -e "s|registry.redhat.io/rhceph/rhceph-5-rhel8:latest|cp.icr.io/cp/ibm-ceph/ceph-5-rhel8:latest|" \
```

Comment:
We'll need to update these for RHCS 6 and RHEL 9. See `src/pybind/mgr/cephadm/module.py` in `ceph-6.1-rhel-patches`

I'll get the newer values to you here by tomorrow.